src/generator/AutoRest.CSharp/Templates/ProjectJsonTemplate.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@using System.Linq
+@inherits AutoRest.Core.Template<AutoRest.CSharp.TemplateModels.ProjectJsonModel>
+{
+  "version": "@Model.Version",
+  "description": "@Model.Description",
+  "authors": [ "Microsoft" ],
+
+  "packOptions": {
+    "summary": "@Model.Description",
+    "tags": [],
+    "requireLicenseAcceptance": true
+  },
+
+  "buildOptions": {
+    "delaySign": true,
+    "publicSign": false,
+    "keyFile": "../../../../tools/MSSharedLibKey.snk"
```

Comment:
are you sure we want these in the vanilla c# version? That path is very specific ...